### PR TITLE
Consistency cleanup

### PR DESCRIPTION
**Pull request by liamjwang**

### CODE REVIEW

File: src/main/java/org/team1540/robot2018/OI.java [hunk 1]
```diff
@@ -43,119 +43,137 @@ private static double scale(double input, double pow) {
...
+  // static Button copilotX = new JoystickButton(copilot, X);
 
-  static Button wristJoystickActivation = new Button() {
+  // ELEVATOR
+  public static double getElevatorAxis() {
+    return scale(Utilities.processDeadzone(copilot.getRawAxis(LEFT_Y), Tuning.axisDeadzone), 2);
+  }
+
+  // TODO: Add a deadzone button to ROOSTER
+  // TODO: Add something (function/interface) for disabling a command when joystick is not being used
```
Comment (RobinsonZ) [inline src/main/java/org/team1540/robot2018/OI.java:75]: Probably should create issues for these.

Comment (liamjwang) [reply src/main/java/org/team1540/robot2018/OI.java:75]: 2fancy4me
ok

File: src/main/java/org/team1540/robot2018/OI.java [hunk 2]
```diff
@@ -43,119 +43,137 @@ private static double scale(double input, double pow) {
...
+  static Button enableElevatorAxisControlButton = new Button() {
+    // Button is pressed when the specified axis is not within the deadzone
     @Override
     public boolean get() {
-      return Utilities.processDeadzone(copilot.getRawAxis(5), Tuning.joystickWristLiftDeadzone)
+      return scale(Utilities.processDeadzone(copilot.getRawAxis(LEFT_Y), Tuning.axisWristLiftDeadzone), 2)
```
Comment (RobinsonZ) [inline src/main/java/org/team1540/robot2018/OI.java]: Why are we bothering to scale the thing if we're just checking if it's 0?

File: src/main/java/org/team1540/robot2018/OI.java [hunk 3]
```diff
@@ -43,119 +43,137 @@ private static double scale(double input, double pow) {
...
+
+  static Button changeWristToTurretButton = new AxisButton(copilot, Tuning.axisDeadzone, LEFT_TRIG);
 
-  static Button copilotRightTriggerSmallPress = new Button() {
+  static Button enableWristOrTurretAxisControlButton = new Button() {
+    // Button is pressed when the specified axis is not within the deadzone
     @Override
     public boolean get() {
-      return getCopilotRightTrigger() > Tuning.joystickDeadzone && getCopilotRightTrigger() < 0.5;
+      return scale(Utilities.processDeadzone(copilot.getRawAxis(RIGHT_Y), Tuning.axisWristLiftDeadzone), 2)
```
Comment (RobinsonZ) [inline src/main/java/org/team1540/robot2018/OI.java]: Again

File: src/main/java/org/team1540/robot2018/OI.java [hunk 4]
```diff
@@ -43,119 +43,137 @@ private static double scale(double input, double pow) {
...
-  public static double getDriverRightY() {
-    return scale(Utilities.processDeadzone(driver.getRawAxis(5), Tuning.joystickDeadzone), 2);
+  public static double getTankdriveForwardsAxis() {
+    return scale(Utilities.processDeadzone(driver.getRawAxis(RIGHT_TRIG), Tuning.axisDeadzone), 2);
   }
 
-  public static double getCopilotRightY() {
-    return scale(Utilities.processDeadzone(copilot.getRawAxis(5), Tuning.joystickDeadzone), 2);
+  // WINCH
+  public static double getWinchInAxis() {
```
Comment (RobinsonZ) [inline src/main/java/org/team1540/robot2018/OI.java:142]: Is this used?

File: src/main/java/org/team1540/robot2018/subsystems/IntakeArms.java [hunk 5]
```diff
@@ -7,8 +7,8 @@
 
 public class IntakeArms extends Subsystem {
 
-  private ChickenTalon arm1 = new ChickenTalon(RobotMap.INTAKE_ARM_1);
-  private ChickenTalon arm2 = new ChickenTalon(RobotMap.INTAKE_ARM_2);
+  private ChickenTalon arm1 = new ChickenTalon(RobotMap.INTAKE_ARM_A);
+  private ChickenTalon arm2 = new ChickenTalon(RobotMap.INTAKE_ARM_B);
```
Comment (RobinsonZ) [inline src/main/java/org/team1540/robot2018/subsystems/IntakeArms.java]: Rename to `armA` and `armB`?

**Review by RobinsonZ — CHANGES_REQUESTED** [review]
A few minor nitpicks.

File: src/main/java/org/team1540/robot2018/OI.java (hunk 4 above)
Comment (liamjwang) [reply src/main/java/org/team1540/robot2018/OI.java:142]: Yes, see lines 152 and 159 of OI

**Review by RobinsonZ — APPROVED** [review]

**Pull request merged**